docker-compose.yml

```diff
@@ -12,7 +12,7 @@ services:
     depends_on:
       - db
   db:
-    build: https://github.com/genome-nexus/genome-nexus-importer.git#b056620a52368edb1c3baa6cec47cd5d3167a782
+    build: https://github.com/genome-nexus/genome-nexus-importer.git#83ae860fd388b51198ecc73e2f572a3b7e1f6a07
```

Comment:
@sandertan this doesn't work i believe, can only refer to branches e.g. https://github.com/thehyve/genome-nexus-importer.git#add_UTRs

Reply:
@inodb Hmm I think it works in my instance after a:
```
mvn  -DskipTests clean install
docker compose up --build -d
```

<img width="644" alt="screen shot 2018-06-21 at 13 56 50" src="https://user-images.githubusercontent.com/9624990/41717808-4125df40-755b-11e8-95c1-0f1a74ace0cf.png">